internal/cli/cli.go

```diff
@@ -176,12 +178,28 @@ func (c *banzaiCli) HTTPTransport() *http.Transport {
 			tls.VerifyPeerCertificate = makeFingerprintVerifier(fingerprintBytes)
 		}
 
-		return &http.Transport{
+		roundTriper = &http.Transport{
 			TLSClientConfig: tls,
 		}
 	}
 
-	return http.DefaultTransport.(*http.Transport)
+	return curlRoundTripper{base: roundTriper, insecureSkipVerify: skip}
+}
+
+type curlRoundTripper struct {
+	base               http.RoundTripper
+	insecureSkipVerify bool
+}
+
+func (c curlRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
+	command, err := utils.GetCurlCommand(r, c.insecureSkipVerify)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Debugf("\n%s\n\n", command.String())
```

Comment:
```suggestion
	log.Debug(command.String())
```

I'd keep the formatting to the formatter